cmd/common/frontends/console/console.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2022-2023 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package console
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+
+	log "github.com/sirupsen/logrus"
+	"golang.org/x/term"
+
+	"github.com/inspektor-gadget/inspektor-gadget/cmd/common/frontends"
+	commonutils "github.com/inspektor-gadget/inspektor-gadget/cmd/common/utils"
+	"github.com/inspektor-gadget/inspektor-gadget/internal/logger"
+)
+
+type Frontend struct{}
```

Comment:
Yeah, in this case it's probably a good choice, because the instance can be assigned to an interface. If that weren't the case, I'd probably still export it, to be able to define a variable holding it / embed it into other structs, etc.